fern/pages/docs/getting-started/global-configuration.mdx

```diff
@@ -303,3 +329,42 @@ layout:
   and a 1px border will separate the sidebar from the content.
 </ParamField>
 
+## Edit this page configuration
+```yaml docs.yml
```

Comment:
can we highlight lines 2 to the end

Reply:
Going to hold off since the rest of the code blocks aren't highlighted.